.travis.yml

```diff
@@ -10,7 +10,9 @@ env:
   - TOX_ENV=py36
   - TOX_ENV=py36-raw
 install:
+  - pip install -U pip
   - pip install tox
+  - pip install -U virtualenv
 before_script:
   - echo "Pull request ${TRAVIS_PULL_REQUEST}"
   - sudo sed -i "s/fsync/#fsync/" /etc/postgresql/9.5/main/postgresql.conf
```

Comment:
Aren't we getting rid of travis?

Reply:
That's the goal, but I want a few PRs with both to compare and make sure they're doing the same stuff.